matchms/importing/__init__.py

```diff
@@ -10,8 +10,12 @@
 
 For more extensive import options we recommend building custom importers using `pyteomics
 <https://github.com/levitsky/pyteomics>`_ or `pymzml <https://github.com/pymzml/pymzML>`_.
+
+To process spectrum metadata, matchms can also make use of known adduct information
+which is imported via :meth:`~matchms.importing.load_adducts`.
```

Comment:
Now it renders `load_adducts()` while load_adducts is a module with to methods.
```suggestion
which is imported via :mod:`~matchms.importing.load_adducts`.
```